app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,34 @@
+class CommentsController < ApplicationController
+
```

Comment:
em không để ak , bởi em chỉ cho phép ng dùng không có tài khoản , chỉ xem khi mà họ bấm vào vào cái nào cần chức năng log in thì nó sẽ trả về trường log in luôn ak